samples/Samples.Dapper/Program.cs

```diff
@@ -0,0 +1,119 @@
+#if !NET452
+
+using System;
+using System.Data;
+using System.Data.Common;
+using System.Threading.Tasks;
+using Dapper;
+using Npgsql;
+using Datadog.Trace;
+
+namespace Samples.Dapper
+{
+    internal static class Program
+    {
+        private static async Task Main()
+        {
+            using (var connection = CreateConnection())
+            {
+                var testQueries = new DapperTestHarness<IDbConnection, IDbCommand, IDataReader>(connection);
+                await testQueries.RunAsync();
+            }
+        }
+
+        private static NpgsqlConnection CreateConnection()
+        {
+            var connectionString = Environment.GetEnvironmentVariable("POSTGRES_CONNECTION_STRING");
+
+            if (connectionString == null)
+            {
+                var host = Environment.GetEnvironmentVariable("POSTGRES_HOST") ?? "localhost";
+                connectionString = $"Host={host};Username=postgres;Password=postgres;Database=postgres";
+            }
+
+            return new NpgsqlConnection(connectionString);
+        }
+    }
+
+    public class DapperTestHarness<TConnection, TCommand, TDataReader>
```

Comment:
What do you think about moving this `DapperTestHarness` class to the `Samples.DatabaseHelper` library, like the `RelationalDatabaseTestHarness` class? I'm thinking that we could run Dapper over all the other database clients as well to make sure we still trace those, though that work could be separated out into a different PR

Reply:
Yes, I think it is better there; meant to re-visit that!